server/controllers/splus.ts

```diff
@@ -53,6 +57,7 @@ router.get('/:schedule/:week', cors(), async (req, res) => {
 
     return lectures.map((lecture) => {
       const richLecture = {
+        id: lectureToId(lecture),
```

Comment:
Das wurde in verschiedenen Teilen des Frontends genutzt. Im ersten Ansatz habe ich dafür eine Bibliothek geschrieben, letztendlich habe ich es in den Server verschoben.